<launch>
    <rosparam command="load" file="$(find ros_phoenix)/launch/box_testing.yaml" />
    <node pkg="ros_phoenix" type="ros_phoenix_node" name="ros_phoenix" respawn="false" />
    <node pkg="ros_phoenix" type="box_testing.py" name="box_testing" respawn="false" output="log">
        <param name="current" type="double" value="10.0" />
    </node>
</launch>